### PR TITLE
Only install Flux v1 for non-migrated envs

### PR DESCRIPTION
**Pull request by zc-hmcts**
### JIRA link ###
https://tools.hmcts.net/jira/browse/DTSPO-5953

### Change description ###
Update bootstrap script to only install Flux v1 for non-migrated envs

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: bootstrap/deploy-flux.sh [hunk 1]
```diff
@@ -198,4 +192,15 @@ if [[ " ${FLUX_V2_CLUSTERS[*]} " =~ " ${ENVIRONMENT} " ]]; then
     flux_v2_pod_identity_sops_setup
     flux_v2_ssh_git_key
     flux_v2_installation
+fi
+
+FLUX_V1_CLUSTERS=( 'dev' 'demo' 'ithc' 'prod' 'ptlsbox' 'stg' 'test' )
```
Comment (msl8r) [inline bootstrap/deploy-flux.sh:197]: has ptl been migrated to v2?

File: bootstrap/deploy-flux.sh [hunk 2]
```diff
@@ -175,17 +175,11 @@ EOF
...
 sleep 60
 helm_add_repo
 echo "****  repo added ****"
 helm_apply_crd ${HELM_OPERATOR_VER}
-flux_ssh_git_key
-echo "****  ssh key added ****"
-flux_install ${flux_repo_list} ${ENVIRONMENT} v3
-echo "****  flux is now installed ****"
 flux_helm_operator_install
 echo "**** helm operator is now installed ****"
```
Comment (msl8r) [inline bootstrap/deploy-flux.sh:185]: Think these lines need to also be moved into the v1 if statement as all the v2 components are installed with flux_v2_installation

Comment (zc-hmcts) [reply bootstrap/deploy-flux.sh:185]: that is what I did

**Review by timja — APPROVED** [review]
Can you also destroy a cluster and rebuild from scratch to make sure this works end 2 end?

**Pull request merged**